crates/kitsune_p2p/bootstrap/Cargo.toml

```diff
@@ -14,7 +14,7 @@ edition = "2018"
 
```

Comment:
how come this crate doesn't get a `-dev` version?

Reply:
I see, it must not have been released.

Reply:
yes, I used the following filter to select the top-level crates for release:

```
--selection-filter="^(holochain|holochain_cli|kitsune_p2p_proxy)$" 
```

only crates matching this and their dependencies were considered for this release.